### PR TITLE
Fix bug 1132092: Fix boolean radio fields for newsletter form.

### PR DESCRIPTION
**Pull request by pmclanahan**

### CODE REVIEW

**Comment by pmclanahan** [comment]
@jgmize or @kyoshino r?

File: bedrock/newsletter/views.py [hunk 1]
```diff
@@ -279,6 +279,7 @@ def existing(request, token=None):
         form = ManageSubscriptionsForm(
             locale, initial=user
         )
+        print initial
```
Comment (jgmize) [inline bedrock/newsletter/views.py]: Looks like you accidentally left this in.

**Comment by jgmize** [comment]
This lgtm other than the print statement. r+wc

**Comment by pmclanahan** [comment]
DOH! Thanks.

**Comment by pmclanahan** [comment]
Fixed.

**Pull request merged**